azure/datalake/store/lib.py

```diff
@@ -89,13 +88,10 @@ def auth(tenant_id=None, username=None,
         indicates this authentication attempt requires two-factor authentication
     authority: string
         The full URI of the authentication authority to authenticate against (such as https://login.microsoftonline.com/)
-    kwargs : key/values
```

Comment:
We are still accepting this param so let's keep it in description as well.
But since we are not using it, we can say: Other parameters, for future use.